src/seedu/addressbook/data/person/Person.java

```diff
@@ -66,6 +71,14 @@ public void setTags(UniqueTagList replacement) {
         tags.setTags(replacement);
     }
 
+    public int getSequenceNumber() {
```

Comment:
good that you've included a getter for this. but do think if a setter should really be there?